manifests/cf-manifest/deployments/040-cf-properties.yml

```diff
@@ -2,9 +2,9 @@ meta:
   environment: ~
   default_quota_definitions:
     default:
-       memory_limit: 10240
-       total_services: 100
-       non_basic_services_allowed: true
+       memory_limit: 2048
+       total_services: 10
+       non_basic_services_allowed: false
```

Comment:
Ah, after re-reading it and thinking about it, I now understand. I hadn't grokked the distinction between:
- limit to 10 basic services which are free
- limit to 10 services which can only be basic/free

I think the updated commit now describes this better :+1: